java/build/bazel/tests/integration/LineListOutputStream.java

```diff
@@ -67,7 +66,8 @@ public void close() throws IOException {
   /**
    * Returns the list of selected lines.
    */
-  ImmutableList<String> getLines() {
-    return ImmutableList.copyOf(lines);
+  
+ List<String> getLines() {
```

Comment:
indentation

Reply:
vscode doesn't show whitespace diff by default. changed it...